lib/v2/screens/explore_screens/vote_screens/proposal_details/interactor/mappers/next_proposal_data_state_mapper.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:seeds/v2/datasource/remote/model/profile_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/v2/screens/explore_screens/vote_screens/proposal_details/interactor/viewmodels/page_commands.dart';
+import '../viewmodels/proposal_details_state.dart';
+
+class NextProposalDataStateMapper extends StateMapper {
+  ProposalDetailsState mapResultsToState(ProposalDetailsState currentState, List<Result> results) {
+    if (areAllResultsError(results)) {
+      return currentState.copyWith(pageState: PageState.failure, errorMessage: 'Error Loading Page');
+    } else {
+      print('ProposalDetailsDataStateMapper mapResultsToState length=' + results.length.toString());
```

Comment:
is this a debugging print?

Reply:
Its has been a pattern a follow for mapper that recibe a List<Result> type. I think was you who introduce it.

Reply:
yeah, i probably left it behind by mistake when i was testing. 
I dont think we need it here. 
:P

Reply:
Ok I'll remove this should I remove the others?

Reply:
yes pls

Reply:
done.